src/transforms/log_to_metric.rs

```diff
@@ -1216,10 +1216,7 @@ mod tests {
         let metric = do_transform(config, event).await.unwrap().into_metric();
         let tags = metric.tags().expect("Metric should have tags");
 
-        assert_eq!(
-            tags.iter_single().collect::<Vec<_>>(),
-            vec![("l1_key1", "val2")]
-        );
```

Comment:
The ordering items inside of the Set created by the toml crate changed and therefore this test started to fail.